Añadidos los peinados de Ume

Sombrahide opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Añadido al codigo todos los peinados de Ume que se encuentran actualmente en el git de hipania paradise.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Tenemos mas peinados disponibles para nuestros personajes
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: Añadido 22 peinados creados por Ume
imageadd: Añadido 22 nuevos iconos creados por Ume
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Sombrahide committed:
Añadidos los peinados de Ume

Añadido al codigo todos los peinados de Ume que se encuentran actualmente en el git de hipania paradise.

Silverexperience commented on code/hispania/modules/sprite_accessories/human/human_hair.dm:
```diff
@@ -0,0 +1,87 @@
+
+/datum/sprite_accessory/hair/low_twins
+	name = "Low Twins"
+	icon_state = "low_twins_s"
```
Al final del código hay que dejar un espacio en blanco.

Sombrahide replied:
Entendido, ahora me pondre a ello

Sombrahide replied:
Espacio en blanco añadido, ademas de haber puesto los pelos de ume en un archivo en la direccion que me recomendastes.

Sombrahide commented:
Sorry, i have pulled all the changes of our code

Silverexperience reviewed (changes requested):
- En el PR de los pelos de ume para que los archivos los pueda leer el programa necesita tener el check activado cuando se compile.

![imagen](https://user-images.githubusercontent.com/58746682/90079162-6e1b1680-dd07-11ea-8b15-19d756c6f70c.png)

- Los pelos de ume seria mejor que los metieras en un archivo dentro de la carpeta icons/hispania y lo reedireccionaras ahi.

Sombrahide committed:
Reajustes al pelo de Ume

Los peinados de Ume ahora se encuentran mejor ordenados, el archivo hair.dmi se encuentra dentro de una nueva carpeta dentro de la carpeta de hispania dentro de la carpeta de icons.
Por otra parte el human_hair.dm no ha cambiado su localizacion, mas si de donde toma los peinados.

Evankhell561 commented on code/hispania/modules/sprite_accessories/human/human_hair.dm:
```diff
@@ -0,0 +1,87 @@
+//Human Hairstyles made by Ume
+/datum/sprite_accessory/hair
```
no existe esto ya en el codigo?
tengo dudas sobre esta linea. Si esta es la ruta de los cabellos del repo principal y cambias al tura entonces los peinados de la rama principal no se visualizarán.

Silverexperience replied:
Ambros fijate en el código de los pelos de los kidans. 
Ahi se ve lo que dice Evan.

Sombrahide replied:
Se que en lo de los kidan tienen una ruta diferente, mas tenerla igual que la del codigo principal no la afecta, los peinados de la rama principal se siguen visualizando, es algo que comprobe antes de hacer el pr mientras comprovaba que los peinados de ume se viesen.

Evankhell561 replied:
No, no está bien. Lo que pasa es que metiste los cabellos de ume en el dmi de la rama princial y aun no los has borrado.

Evankhell561 reviewed (changes requested):
También, por qué has modificado dos archivos .dmi?

Sombrahide committed:
Añadido dme y asegurado que el dmi de pelo por defecto este como toca

Sombrahide commented:
> 
> 
> También, por qué has modificado dos archivos .dmi?

Se debe a que antes los peinados de ume estaban en la rama principal, pero Silver de dijo que los cambiase a una aparte, cosa que he echo y por supuesto tuve que quitarlos de la principal.

Sombrahide committed:
Finalmente los peinados estan en su sitio

Sombrahide committed:
Peinados Ume, la ultima version?

Sombrahide committed:
Cambio de nombres y añadido del peinado perdido

Sombrahide committed:
El alternativo que escapo
